lib/net/ldap.rb

```diff
@@ -1331,12 +1344,35 @@ def new_connection
   # Normalize encryption parameter the constructor accepts, expands a few
   # convenience symbols into recognizable hashes
   def normalize_encryption(args)
-    return if args.nil?
-    return args if args.is_a? Hash
+    valid_args =
+      "encryption may be hash, nil, or one of [:simple_tls, :start_tls, true]"
```

Comment:
Down the road, I'm thinking of splitting `tls_options` out as a top-level argument passed into `Net::LDAP.new`. `@encryption` should then be able to be just a symbol. Having it be a hash with two entries makes it more complicated to invoke correctly.